faker/providers/python/__init__.py

```diff
@@ -181,32 +191,67 @@ def pyfloat(
                 f"{sys.float_info.dig})",
             )
 
-        sign = ""
         if (min_value is not None) or (max_value is not None):
             # Copy values to ensure we're not modifying the original values and thus going out of bounds
-            left_min_value = min_value
-            left_max_value = max_value
+            temp_min_value = min_value
+            temp_max_value = max_value
```

Comment:
Renamed to `temp` rather than `left` because it is now doing both the right and left parts.